.gitignore

```diff
@@ -1,54 +1,35 @@
-# --------------------
-# OSX Files
-# --------------------
+### SublimeText ###
+*.sublime-workspace
+
+### OSX ###
 .DS_Store
 .AppleDouble
 .LSOverride
 Icon
+
+# Thumbnails
 ._*
+
+# Files that might appear on external disk
 .Spotlight-V100
 .Trashes
 
-# --------------------
```

Comment:
Why are you removing so much of the gitignore? This is the same file we use elsewhere and it is a good baseline for multiple environments both windows and mac.